src/app/tasks/checker-inbox-and-tasks-tabs/checker-inbox/checker-inbox.component.html

```
@@ -6,34 +6,33 @@
-      </div>
-    </div>
+<div class="tab-container mat-typography">
+  <div style="display: flex; align-items: center; gap: 8px; margin-bottom: 16px">
+    <mat-form-field style="width: 280px; margin: 0">
+      <input
+        matInput
+        placeholder="{{ 'labels.text.Search by user' | translate }}"
+        (keyup)="applyFilter($event.target.value)"
+      />
+    </mat-form-field>
+    <button mat-icon-button (click)="toggleAdvancedSearch()" style="margin: 0">
```

HTMLHint (1.8.0)

[warning] Inline style [  style="margin: 0" ] cannot be used.

(inline-style-disabled)